allocations/allocations_api.py

```diff
@@ -0,0 +1,31 @@
+import requests
+import traceback
+from django.conf import settings
+import logging
+from keystoneauth1.identity import v3
+from keystoneauth1 import session
+from project.view import get_ks_auth_and_session
+
+logger = logging.getLogger('allocations')
+
+class BalanceServiceClient:
```

Comment:
Where is this used? I didn't see any reference to it.

I'd also recommend doing a bit of checking in an `__init__()` function to see if the settings are set up. It should raise an error if there is no `ALLOCATIONS_BALANCE_SERVICE_ROOT_URL`, as that looks to be required.